src/test/java/com/redhat/red/build/koji/model/json/KojiImportTest.java

```diff
@@ -43,17 +30,16 @@
 public class KojiImportTest
         extends AbstractJsonTest
 {
-
     @Test
     public void jsonRoundTrip()
             throws VerificationException, IOException
     {
         KojiImport info = new KojiImport( KojiJsonConstants.DEFAULT_METADATA_VERSION, newBuildDescription(),
-                                          Collections.singleton( newBuildRoot() ),
+                                          Arrays.asList( new BuildRoot[] { newBuildRoot() } ),
```

Comment:
I think Arrays.asList is a varargs method, so you probably don't need the `new BuildRoot[]{}` wrapper...